docs/IntegrationWithExistingApps.md

```diff
@@ -618,6 +624,7 @@ public class MyReactActivity extends Activity implements DefaultHardwareBackBtnH
                 .addPackage(new MainReactPackage())
                 .setUseDeveloperSupport(BuildConfig.DEBUG)
                 .setInitialLifecycleState(LifecycleState.RESUMED)
+                //.setUseOldBridge(true) // uncomment this line if your app crashes
```

Comment:
Why would the app crash? Maybe provide an example reason in the comment? Or add a note above this code about possible reasons for crashing.